.circleci/config.yml

```
@@ -906,18 +906,14 @@ jobs:
           name: Linting project
```

question: is there any reason we can't/shouldn't move the code-analysis (or at least the prodsec checks part of it) to the start of the CICD job, in parallel with the `secrets-scan`. This was we can fail fast.

---

This sounds like a good follow up PR, I think there are loads of interesting changes we could make to the CI/CD pipeline to speed up the feedback loop.